packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -218,15 +300,18 @@ export default decorate([
         />
         <br />
       </CardHeader>
-      <CardBlock className='text-center'>
-        <div>
-          <span className='text-muted'>{_('os')}</span> <strong>{os}</strong>
-        </div>
-        <div>
-          <span className='text-muted'>{_('version')}</span>
-          {'  '}
-          <strong>{version}</strong>
-        </div>
+      <CardBlock>
+        <div
+          className='text-muted'
+          dangerouslySetInnerHTML={state.description}
+        />
+        <Button
+          className='pull-right'
+          onClick={effects.showDescription}
+          size='small'
+        >
+          <Icon icon='info' /> {_('moreDetails')}
```

Comment:
Can't you use an `ActionButton` here?

Reply:
Do you want to use an `actionButton` just to avoid adding an Icon ?

Reply:
Yes, and to keep it maintainable and homogeneous with the other buttons.